shortfin/python/shortfin_apps/sd/server.py

```diff
@@ -158,18 +159,19 @@ def get_configs(args):
                 arglist = spec.strip("--").split("=")
                 arg = arglist[0]
                 if len(arglist) > 2:
+                    print(arglist)
                     value = arglist[1:]
                     for val in value:
                         try:
                             val = int(val)
                         except ValueError:
-                            continue
+                            val = val
```

Comment:
Why are you doing val=val and value=value below?

Reply:
This is mostly for the device ids arg, where user can pass either device indexes (ints) or device UIDs (strings).
The logic being: if we can't convert them to ints, then we pass them as they were recieved.
doing _continue_ here breaks parsing of non-list/int arguments e.g. `isolation`.